### PR TITLE
Added ng-katex link to readme.md

### PR DESCRIPTION
**Pull request by garciparedes**
From issue https://github.com/Khan/KaTeX/issues/688

### CODE REVIEW

**Comment by garciparedes** [comment]
[clabot:check]

File: README.md [hunk 1]
```diff
@@ -86,6 +86,12 @@ the same aspect ratio in KaTeX as in LaTeX.  However, because most browsers
 default to a larger font size, this typically means that a 1cm kern in KaTeX
 will appear larger than 1cm in browser units.
 
+## Libraries
```
Comment (kevinbarabash) [inline README.md]: Thanks for adding this section.  I'm sure we'll be expand this to include other libraries/components.

**Comment by kevinbarabash** [comment]
@garciparedes thanks for the PR.

File: README.md (hunk 1 above)
Comment (ronkok) [reply README.md]: Perhaps [katex-ruby](https://github.com/glebm/katex-ruby) from @glebm would be a good addition.

Comment (edemaine) [reply README.md]: Agreed. Also, wasn't there a KA plugin of some relevance?

**Pull request merged**